addon/templates/components/big-text.hbs

```diff
@@ -1,8 +1,16 @@
-<span class={{if renderHtml "list-reset"}}>
+<span
+  class={{if renderHtml "list-reset"}}
```

Comment:
missing `class='clickable'` which is required in some browsers to take the click on a span. These should really be buttons, but that's for another day.